src/render/PanelHeaderProvider.js

```diff
@@ -185,7 +185,8 @@ function getTemplatesService() {
 
 function getTemplate(element, elementTemplates) {
   const templateId = elementTemplates._getTemplateId(element);
-  return templateId && elementTemplates.get(templateId);
+  const templateVersion = elementTemplates._getTemplateVersion(element);
+  return templateId && elementTemplates.get(templateId, templateVersion);
```

Comment:
Why not use `elementTemplates.get(element)` instead? It supports elements as arguments

https://github.com/bpmn-io/bpmn-js-properties-panel/blob/d96930011cec7e5db04776a375fc0e4f41544878/src/provider/element-templates/ElementTemplates.js#L26-L33

Reply:
Good catch!

Reply:
I adjusted it accordingly 👍  Thanks!